lua/various-textobjs/linewise-textobjs.lua

```diff
@@ -141,7 +141,9 @@ end
 ---indentation textobj
 ---@param startBorder "inner"|"outer"
 ---@param endBorder "inner"|"outer"
-function M.indentation(startBorder, endBorder)
+---@param includeBlankLines? "true"|"false"
```

Comment:
well, not string-enum like that 😅 

a name `blankLines` with the possibilities `"withBlanks"|"noBlanks"` is what I meant.

Then your keymap looks like this:
```lua
require("various-textobjs").indentation("outer", "inner", "noBlanks")
require("various-textobjs").indentation("outer", "inner", "withBlanks")
```
which is more verbose, as you can see at a glance what each mapping does.

Reply:
Ok, I changed to an enum, but I left the internal variable as a boolean because I think this reads better:

`((includeBlankLines and isBlankLine(prevLnum)) or fn.indent(prevLnum) >= indentOfStart)`